config/config.go

```diff
@@ -45,6 +47,7 @@ type Config struct {
 	AcceptanceFiles      []string
 	AcceptanceExtensions []string
 	NoColor              bool
+	Platform             string
```

Comment:
Should it be set from CLI? Don't see that it's assigned

Reply:
We can leave this.

Reply:
saw it was removed from config